mk/lib.mk

```diff
@@ -122,14 +122,15 @@ $(foreach script, $(bin-scripts), $(eval $(call install-program-in,$(script),$(b
 $(foreach script, $(bin-scripts), $(eval programs-list += $(script)))
 $(foreach script, $(noinst-scripts), $(eval programs-list += $(script)))
 $(foreach template, $(template-files), $(eval $(call instantiate-template,$(template))))
+install_test_init=tests/functional/init.sh
```

Comment:
This is a layer violation, but it is not a new layer violation. `mk` previously also new about `init.sh`. The violation is in fact "shallower" (closer to the entry point) then before. That seems like an improvement to me.